corehq/motech/models.py

```diff
@@ -31,10 +31,11 @@ class Meta:
     @staticmethod
     def unpack_request_args(request_method, args, kwargs):
         params = kwargs.pop('params', '')
-        json_data = kwargs.pop('json', '')
-        data = kwargs.pop('data', json_data)  # Don't bother trying to
-        # cast as a dict. `data` will be rendered as prettified JSON if
-        # possible, regardless of whether it's a dict or a string here.
+        json_data = kwargs.pop('json', None) or ''
+        data = kwargs.pop('data', None) or json_data  # Don't bother
```

Comment:
What is the benefit changing this? are `data` or `json` sometimes passed as false-y values?

Reply:
[`Session.post()` passes them to `Session.request()`](https://github.com/kennethreitz/requests/blob/master/requests/sessions.py#L581) with default values of `None`, and I wanted to keep the behaviour the same. I also wanted to ensure that if `json` is `None` (because it wasn't passed and got its default value) then `data` would still end up as an empty string, to avoid it being rendered in logs as "None". ... 

But I see where you're going with this. If `data` or `json` are falsey, then `data` will end up with the wrong value. What's more, this is not the right place to decide what value gets rendered.

I'll fix it.